automol/form/_form.py

```diff
@@ -173,7 +174,7 @@ def string(fml: dict[str:int], hyd: bool = True) -> str:
     return fml_str
 
 
-def string2(fml: dict[str:int]) -> str:
+def string2(fml: Formul) -> str:
     """Convert formula dictionary to formula string that includes 1s in when
     there is only one atom.
 
```

Comment:
Looks good up to this point. A few more edits for the functions below this point:
```
def from_string(...) -> Formula:
...
def sorted_symbols(...) -> tuple[str, ...]:
...
def sort_vector(fml: Formula, symbs: Sequence[str] | None = None) -> tuple[int, ...]:
```